proposal_stages.md

```diff
@@ -0,0 +1,28 @@
+# ROS 2 Production Proposal Stages
+The following stages are also intended to be used as github tags for tracking.
+## Stage 1: Draft
+
+The draft stage represents the initial proposal of a project or task which has been formalised into a github issue for presentation to the community. To move to the next stage, this draft should be presented to the ROS2 Production WG and discussed in relevant channels (e.g slack, github, discourse, etc.), having addressed feedback about the scope, methodology, and tasks required to implement this proposal. 
+
+## Stage 2: Ready
+
+In this stage, the draft has received feedback from the community. It is considered to be in “ready for work” state. Tasks have been clearly defined and broken down into relevant sub-components and subtasks, sufficient such that a principal contributor may begin execution with clear direction. To move to the next stage, there should be a discussion/vote on the proposal’s priority, who will be the principal contributor, and the intended timeline for completion. 
+
+
+## Stage 3: Accepted
+
+In this stage, the proposal has been accepted and is part of the ROS 2 Production WG’s project planning board. A principal contributor has been identified. Accepted proposals will be visible on the working group’s project roadmap. To move to the next stage, the principal contributor should be providing regular updates to the working group. 
+
+## Stage 4: In Progress
+
+In this stage, the proposal is in the middle of development and is receiving updates regularly from contributors. For in-progress proposals, they will be updated to the wider technical steering committee. To move to the next stage, the proposal work should be near completion and receiving final reviews.
+
+## Stage 5: In Review
+
+In this change, other community members/developers are reviewing the implementation and quality of the proposal/task work, with minor* changes, if any, that may be required for completion. To move to the next stage, all issues from reviewers have been addressed.
+
```

Comment:
Should we have a testing step ?

Reply:
Agree - or it can be part of the review
* Get multiple companies to try the feature out
* Ensure it runs reliably in CI if it's something that is automated (i.e - it's not flaky)
* Ensure the documentation is sufficient for other companies to use the feature

Reply:
A common occurence I often see is that all "tests" may be passing, however the work may be still be blocked by reviewers based on how it is implemented. Perhaps we can fold testing requirements, CI, documentation etc. as part of the criteria for being considered `Done`?